src/getID3/Tags/getid3_id3v2.php

```diff
@@ -1306,7 +1301,7 @@ public function ParseID3v2Frame(&$parsedFrame) {
 			$parsedFrame['feedbackRR']    = ord(substr($parsedFrame['data'], $frame_offset++, 1));
 			$parsedFrame['feedbackRL']    = ord(substr($parsedFrame['data'], $frame_offset++, 1));
 			$parsedFrame['premixLR']      = ord(substr($parsedFrame['data'], $frame_offset++, 1));
-			$parsedFrame['premixRL']      = ord(substr($parsedFrame['data'], $frame_offset++, 1));
+			$parsedFrame['premixRL']      = ord(substr($parsedFrame['data'], $frame_offset, 1));
```

Comment:
for consistency it might make sense to leave the `++`, e.g. if a line is added below, but it's fine semantically here, so :+1:

Reply:
yes, I thought that also. The IDE shouting is a little bit silly, because if someone adds another line of code below here, it will be easy to forget to adjust the line above.